modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/grid/GridDDMFormFieldValueRenderer.java

```diff
@@ -103,8 +104,11 @@ protected String getLabel(
 		return defaultLabel;
 	}
 
-	@Reference
-	protected GridDDMFormFieldValueAccessor gridDDMFormFieldValueAccessor;
+	@Reference(
+		target = "(component.name=com.liferay.dynamic.data.mapping.form.field.type.internal.grid.GridDDMFormFieldValueAccessor)"
```

Comment:
@RenanUmV You can also use `ddm.form.field.type.name=...` for the target. Both are valid, but I think `ddm.form.field.type.name` is more semantically clear.

cc @rafaprax

Reply:
@RenanUmV @rafaprax For this one, it seems all the previous pulls are just using `component.name`? If so, no need to change it to use the property I mention. Just keep your current way to be consistent.